setup/01-manual_testing.py

```diff
@@ -1,46 +1,30 @@
 import os
 import sys
-import re
 from static import LOGIN_ROBOT_FILE
 
 def check_file_exists():
     if not os.path.isfile(LOGIN_ROBOT_FILE):
-        print(f"login.robot not found under {LOGIN_ROBOT_FILE}, please create one")
+        print(f"Hmm. I don't see your 'login.robot' file. I was expecting to see it at, '{LOGIN_ROBOT_FILE}'")
+        print("Double-check that it's in the right place and named correctly and try again.")
         sys.exit(1)
 
-def check_content():
-    is_browser = False
-    is_url = False
-    is_password = False
-    is_username = False
-    is_login = False
-    is_welcome = False
-    data = None
-    with open(LOGIN_ROBOT_FILE, "r") as f:
-        data = f.readlines()
-    for line in data:
-        lowered_line = line.lower()
-        if re.search(r"(new|open) (page|browser)", lowered_line):
-            is_browser = True
-        elif re.search(r"localhost:7272", lowered_line):
-            is_url = True
-        elif "username" in lowered_line:
-            is_username = True
-        elif "password" in lowered_line:
-            is_password = True
-        elif "click" in lowered_line or "login" in lowered_line:
-            is_login = True
-        elif "welcome" in lowered_line:
-            is_welcome = True
-    if is_browser and is_url and is_password and is_username and is_login and is_welcome:
-        print("Ready to proceed!")
-    else:
-        print("Not quite there yet! Did you remember browser, username, password or perhaps missing url? Did you verify your login succeeded?")
-        sys.exit(1)
+def demonstrate_our_solution():
+    print("Nice job!")
+    print("If you'd like to compare answers, here's our solution:")
+    print("""
+    Open browser
+    Navigate to localhost:7272
+    Enter username
+    Enter password
+    Submit login form
+    Check that welcome page text is visible
+    """)
 
 def main():
     check_file_exists()
-    check_content()
```

Comment:
This should not be taken out. We also need to validate that the exercise is written correctly in addition that the file exists.

Reply:
Yes, I agree that the check on the first few exercises is _way_ too strict and I've said that to participants multiple times (which is why I have loosened some of them up as I found them frustrating). And I agree that the printed message isn't helpful at all. Still, I think that even a bad check is better than no check at all.

The verification should be improved, not removed. `rflint` isn't the best in regards of documentation simplicity, which is why there's #27 to investigate how `robocop` could be used instead of `rflint` in terms of rule customization.

The verification of the exercise could be split up to multiple functions, so instead of one big `check_content` we could have `check_open_browser`, `check_enter username`, `check_password`, and so on each function printing an "error message" if the step is missing. Yes, this costs quite a bit of performance as the steps need to be looped multiple times. Other possibility is to have separate `if` statements in the `check_content` for each individual verification and print the "error".